src/components/member-profile/contribution/index.js

```diff
@@ -63,11 +70,13 @@ const ContributionCard = ({
   urlObj,
 }) => {
   const {
-    task: { title, startedOn, endsOn, status, purpose },
+    task: { id, title, startedOn, endsOn, status, purpose, isNoteworthy },
     prList,
   } = contribution;
   const isTitleAvailable = !!title;
   const featureTitle = isTitleAvailable ? title : prList[0].title;
+  const [showSettings, setShowSettings] = useState(false);
+  const { isOptionKey, setIsOptionKey } = useTaskContext();
```

Comment:
This context is going to be changed to another context and then we don't need to take care of the option key for every component separately. Issue for reference⬇️
- https://github.com/Real-Dev-Squad/website-members/issues/420